app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def new
+    redirect_to home_path unless !logged_in?

```

Comment:
Oh I see now, there is another miss use - `unless !` is just `if` never mix this, confused me a lot.